mis_builder_demo/__manifest__.py

```diff
@@ -5,7 +5,7 @@
     "name": "MIS Builder Demo",
     "summary": """
         Demo addon for MIS Builder""",
-    "version": "14.0.3.1.3",
+    "version": "15.0.3.1.0",
```

Comment:
```suggestion
    "version": "15.0.1.0.0",
```

Reply:
@leemannd same here https://github.com/OCA/mis-builder/pull/406#discussion_r794295627

Reply:
@yankinmax You can ignore my comment about the version numbers